4-hangman/hangman02.py

```diff
@@ -0,0 +1,13 @@
+import random
+my_list = ["I", "eat", "chicken", "and", "turkey"]
+random_word = str(random.choice(my_list))
+# print(random_word.upper())
```

Comment:
Keep the print statement for now. For testing purposes

Reply:
Done